lib/liveValidation/operationValidator.ts

```diff
@@ -128,7 +128,12 @@ export const validateSwaggerLiveResponse = async (
     validateContentType(operation.produces!, headers, false, result);
   }
 
-  const ctx = { isResponse: true, includeErrors, statusCode };
+  const ctx = {
+    isResponse: true,
+    includeErrors,
+    statusCode,
+    isGetPutHttpMethods: operation._method === "get" || operation._method === "put",
```

Comment:
How about rename to "httpMethod" to enable more future scenario

Reply:
Refactored.